tests/components/zwave/test_websocket_api.py

```diff
@@ -35,3 +37,13 @@ async def test_zwave_ws_api(hass, mock_openzwave, hass_ws_client):
     assert result[CONF_USB_STICK_PATH] == "/dev/zwave"
     assert not result[CONF_AUTOHEAL]
     assert result[CONF_POLLING_INTERVAL] == 6000
+
+    await client.send_json({ID: 6, TYPE: "zwave/get_migration_config"})
+    msg = await client.receive_json()
+    result = msg["result"]
+
+    assert result[CONF_USB_STICK_PATH] == "/dev/zwave"
+    assert (
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6103615)